### PR TITLE
doc: OpenSSL 3.0.7 not expected to impact Node.js 16.x

### PR DESCRIPTION
**Pull request by arhart**
cc: @juanarbol @mhdawson @RafaelGSS

### CODE REVIEW

File: locale/en/blog/vulnerability/openssl-november-2022.md [hunk 1]
```diff
@@ -23,8 +23,11 @@ the _highest_ severity issue is CRITICAL.
 
 [security policy](https://www.openssl.org/policies/secpolicy.html).
 
-Node.js v16.x, v18.x and v19.x use OpenSSL v3.
-Therefore all active LTS release lines are impacted by this update.
+Node.js v18.x and v19.x use OpenSSL v3.
+Therefore these active LTS release lines are impacted by this update.
+
+Node.js v16.x uses OpenSSL v1.1.1.
+Therefore it is currently NOT expected to be impacted by this update.
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/openssl-november-2022.md:30]:
```suggestion
```

Comment (juanarbol) [inline locale/en/blog/vulnerability/openssl-november-2022.md]: Is this considered a dupe of https://github.com/nodejs/nodejs.org/pull/4901?

Comment (mhdawson) [reply locale/en/blog/vulnerability/openssl-november-2022.md]: Discussing with Andrew in terms of what we want to do between the two. Most likely will use an updated version of this one.

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
@juanarbol just waiting on CI now, then will land unless you have any additional comments/suggestions

**Pull request merged**